src/client/src/App/Trades/TradesGrid/styled.ts

```diff
@@ -8,6 +8,8 @@ export const Table = styled.div`
   border-bottom: 15px solid ${({ theme }) => theme.core.darkBackground};
   height: calc(100% - 4.75rem);
   width: 100%;
+  overflow-x: scroll;
```

Comment:
Do we have to display the scroll constantly, or maybe it is better to be auto-displayed?

Reply:
For some reason, the bar _is_ being auto hidden even though the overflow is not set to auto. I will change the CSS in any case

Reply:
On investigation it seems like the scroll bar is still there but the handle dissapears when the content isn't clipped